Library/Source/NativeEngine.cpp

```diff
@@ -1023,6 +1045,7 @@ namespace babylon
             format,                                         // Self-explanatory
             0x0,                                            // Flags
             allPixels);                                     // Memory
+        return Napi::Value::From(info.Env(), (textureData->Texture.idx != bgfx::kInvalidHandle));
```

Comment:
Probably should do this everywhere
```suggestion
        return Napi::Value::From(info.Env(), isValid(textureData->Texture));
```